### DISCUSSION TITLE
monai.engines Examples

### DISCUSSION THREAD

**Discussion by lhallee** [Project-MONAI/tutorials#1242] [Q&A]
Hi. I found this example of [create_multi_gpu_supervised_trainer()](https://github.com/Project-MONAI/tutorials/blob/4b2771acbd053683854a8559d9dbc59c7c8703a7/acceleration/multi_gpu_test.ipynb) but I would love some more examples of user the create_multi_gpu_supervised_trainer and evaluator together in different scenarios (with actual dataloaders if possible). Could anyone provide a link or some code? Thanks so much.

**Comment by Nic-Ma** [comment]
Hi @lhallee ,

These 2 APIs are deprecated in the latest dev code, please use `SupervisedTrainer` and `SupervisedEvaluator` instead.
Refer to examples: https://github.com/Project-MONAI/tutorials/tree/main/modules/engines.

Thanks.

**Reply by lhallee** [reply]
Thanks so much, Nic, that is very helpful.
